ui/demo/less/reader.less

```diff
@@ -60,32 +60,17 @@ body {
 
 
 .reader__popover__bbox {
-  border: 1px transparent dotted;
+  border: 1px transparent dashed;
 
-  &.rotate0 {
-    border-bottom-color: @dark-grey
-  }
-  
-  &.rotate90 {
-    border-left-color: @dark-grey;
-  }
-
-  &.rotate180 {
-    border-top-color: @dark-grey;
+  &.rotate0, &.rotate90, &.rotate180, &.rotate270 {
+    border-bottom-color: @initial-blue;
   }
 
-  &.rotate270 {
-    border-right-color: @dark-grey;
-  }
-
-  &:hover {
-    background-color: rgba(19, 92, 202, 0.2);
-    cursor: pointer;
-  }
-
-  &.selected {
-    background-color: rgba(19, 92, 202, 0.3);
+  &:hover, &.selected {
+    background-color: @hover-active-blue;
+    mix-blend-mode: screen;
```

Comment:
Is `mix-blend-mode` needed here if it's specified in the hover and selected states in the library CSS?

Reply:
@carolinepaulic it both needed because without `mix-blend-mode` we cant overwrite the color of the citation text

Reply:
Does that mean that whoever imports this library will have to partially rewrite the styling on their end despite us including hover/selected styling in the library? That doesn't seem right.

Reply:
@huytr1995 I think you can remove all these styles and everything would stay the same. And it would make sense to do that since the team decided to put these styles in the lib anyways.